src/index.test.ts

```diff
@@ -1,29 +1,67 @@
-import { getIssueIdFromBranchName } from ".";
-
-describe("getIssueIdFromBranchName", () => {
-
-    test('it should parse the branch name correctly', () => {
-        const branchName = "feature/eng-952-wow-make-pr-titles-autoupdate-with";
-        const issueId = getIssueIdFromBranchName(branchName);
-        expect(issueId).toBe("ENG-952");
-    });
-
-    test('it should ignore other numbers', () => {
-        const branchName = "feature/eng-952-234-wow234-make-pr-3746-2726";
-        const issueId = getIssueIdFromBranchName(branchName);
-        expect(issueId).toBe("ENG-952");
-    });
-
-    test('it should work with other teams', () => {
-        const branchName = "feature/pro-397-something";
-        const issueId = getIssueIdFromBranchName(branchName);
-        expect(issueId).toBe("PRO-397");
-    });
-
-    test('it should return nothing upon incorrect format', () => {
-        const branchName = "nonfeature/eng-397-something";
-        const issueId = getIssueIdFromBranchName(branchName);
-        expect(issueId).toBeNull();
-    });
-
-})
+import { getLinearIssueIds } from ".";
+
+let pullRequest;
+
+beforeEach(() => {
```

Comment:
IMHO there's no need for a `beforeEach` here (and setting a global variable in it is fairly dirty 😅). We can just have this `pullRequest` as a function that takes a branch name and a body and returns this data + `head.ref` set to the branch name and `body` set to the body.

Reply:
Done in https://github.com/perdoo/linear-pr-title-action/pull/2/commits/9ff441d9b382e71b3cdee9fb9cfea0ab16d2598d.